src/install.sh

```diff
@@ -791,15 +811,27 @@ cat <<EOF > ${BUILD_DIR}/pipeline/pipeline.json
           "cluster": "orca"
       },
       "name": "Deploy orca",
-      "refId": "9",
+      "refId": "10",
```

Comment:
`refId`s `"9"` and `"10"` are both taken. I'm surprised this worked end-to-end. Did you redeploy after it was insalled?

Reply:
good catch, thanks. Yeah, it deployed and worked lol